### PR TITLE
VIH-7337 vh booking queue subscriber

### PR DESCRIPTION
**Pull request by lowflying**
### JIRA link (if applicable) ###

N/A

### Change description ###

Added helm charts

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by simonwkelso — APPROVED** [review]
LGTM

File: charts/vh-booking-queue-subscriber/values.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    vh-booking-queue:
+      excludeEnvironmentSuffix: false
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - azuread--clientsecret
+        - azuread--clientid
+  environment:
+    ASPNETCORE_URLS: http://+:8080
+    AZUREAD__TENANTID: "fb6e0e22-0da3-4c35-972a-9d61eb256508"
```
Comment (PrasannaDommalapati) [inline charts/vh-booking-queue-subscriber/values.yaml:30]: Tenant Id hardcoded

**Review by PrasannaDommalapati — CHANGES_REQUESTED** [review]
tenant id is hardcoded

File: charts/vh-booking-queue-subscriber/values.yaml (hunk 1 above)
Comment (lowflying) [reply charts/vh-booking-queue-subscriber/values.yaml:30]: yep, I don't think that's an issue though? Tenant ID's are not secrets

Comment (lowflying) [reply charts/vh-booking-queue-subscriber/values.yaml:30]: Oh, and that's the tenant for AAD

**Review by PrasannaDommalapati — APPROVED** [review]
LGTM

**Pull request merged**